browser-test/src/name.test.ts

```diff
@@ -0,0 +1,270 @@
+import { Page } from 'playwright'
+import {
+  AdminPrograms,
+  AdminQuestions,
+  ApplicantQuestions,
+  loginAsAdmin,
+  loginAsGuest,
+  logout,
+  resetSession,
+  selectApplicantLanguage,
+  startSession,
+} from './support'
+
+const NAME_FIRST = ".cf-name-first";
+const NAME_LAST = ".cf-name-last";
+
+describe('name applicant flow', () => {
+  let pageObject: Page
+
+  beforeAll(async () => {
+    const { page } = await startSession()
+    pageObject = page
+  })
+
+  afterEach(async () => {
+    await resetSession(pageObject)
+  })
+
+  describe('single required name question', () => {
+    let applicantQuestions: ApplicantQuestions
+    const programName = 'test program for single name'
+
+    beforeAll(async () => {
+      await loginAsAdmin(pageObject)
+      const adminQuestions = new AdminQuestions(pageObject)
+      const adminPrograms = new AdminPrograms(pageObject)
+      applicantQuestions = new ApplicantQuestions(pageObject)
+
+      await adminQuestions.addNameQuestion({
+        questionName: 'name-test-q',
+      })
+      await adminPrograms.addAndPublishProgramWithQuestions(
+          ['name-test-q'],
+          programName
+      )
+      debugger;
```

Comment:
what's this I'm not familiar with it?

Likely should remove it though?

Reply:
Yikes, definitely shouldn't have been left in there! 

Maybe I should cut a ticket for lint checks on these test files?

Reply:
Eh, I don't think this is common, I still don't know what it does :D and it didn't seem to affect the test runs.